### PR TITLE
Adding Gradle Test Profiler

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

**Comment by szpak** [comment]
Nice looking plugin. How did you find it? ;)

+1

**Comment by szpak** [comment]
Maybe it would be useful also in micro-infra?

File: build.gradle [hunk 1]
```diff
@@ -142,6 +144,17 @@ task docker(type: Docker) {
     entryPoint(['java', '-jar', "/${project.name}/${project.name}.jar"])
 }
 
+testprofiler {
+    minTestThreshold = 100
+
+    buildBreaker {
+        maxTestThreshold = project.hasProperty('maxTestThreshold') ? project.property('maxTestThreshold') : 30_000
```
Comment (szpak) [inline build.gradle]: `project` is redundant

Comment (szpak) [inline build.gradle]: I have to contribute to Gradle something like:
`getPropertyIsExistOrReturnDefaultValue('maxTestThreshold', 30_000)`

Comment (marcingrzejszczak) [inline build.gradle]: definitely! I've added it to our projects a couple of times and enough is enough :P

**Pull request merged**